assets/javascripts/feature.js

```diff
@@ -1,79 +1,130 @@
-  var tour = new Tour({
-    storage: window.localStorage, // necessary for multipage traversal
-    template: function () {
-      return (
-        "<div class='popover tour'>" +
-        "<div class='arrow'></div>" +
-        "<h3 class='popover-header'></h3>" +
-        "<div class='popover-body'></div>" +
-        "<div class='popover-navigation'>" +
-        "<div class='checkbox'><label><input type='checkbox' id='dont-notify'>Don't notify me anymore (permanent)</label></div>" +
-        "<button class='btn btn-default' data-role='prev' id='tour-prev'>« Prev</button>" +
-        "<button class='btn btn-default' data-role='next'id='tour-next'>Next »</button>" +
-        "<button class='btn btn-default' data-role='end' id='tour-end'>Quit</button>" +
-        '</div>' +
-        '</div>'
-      );
-    },
-    onShown: function (tour) {
-      // allow user to quit the tour at any point and to disable the tour permanently
-      $('#tour-end').on('click', function () {
-        $.ajax({
-          url: '/api/v1/feature',
-          method: 'POST',
-          data: {version: $('#dont-notify').is(':checked') ? 0 : latestFeature}
-        });
-      });
+  const tour = new Shepherd.Tour({
+    defaultStepOptions: {
+      cancelIcon: {
+        enabled: true
+      },
+      confirmCancel: true
     }
   });
-
-  // add steps to the tour
   tour.addSteps([
     {
-      element: '#all_tests',
       title: 'All tests area',
-      content:
-        'In this area all tests are provided and grouped by the current state. You can see which jobs are running, scheduled or finished.',
-      placement: 'bottom',
-      backdrop: false
+      text: "<p>In this area all tests are provided and grouped by the current state. You can see which jobs are running, scheduled or finished.</p><div class='checkbox'><label><input type='checkbox' id='dont-notify' onchange='switchCheckbox(this)'>Don't notify me anymore (permanent)</label></div>",
+      attachTo: {
+        element: '#all_tests',
+        on: 'bottom'
+      },
+      buttons: [
+        {
+          text: 'Prev',
+          action: tour.back,
+          classes: 'btn btn-default '
+        },
+        {
+          text: 'Next',
+          action: tour.next,
+          classes: 'btn btn-default '
+        }
+      ],
+      when: {
+        cancel: function () {
+          postFeature(latestFeature);
+        }
+      }
     },
     {
-      element: '#job_groups',
       title: 'Job group menu',
-      content:
-        'Access the job group overview pages from here. Besides test results, a description and commenting area are provided.',
-      placement: 'bottom',
-      backdrop: false
+      text: "<p>Access the job group overview pages from here. Besides test results, a description and commenting area are provided.</p><div class='checkbox'><label><input type='checkbox' id='dont-notify' onchange='switchCheckbox(this)'> Don't notify me anymore (permanent)</label></div>",
```

Comment:
I'd say this can be inlined and the variable should have a better name:

```suggestion
      text: "<p>Access the job group overview pages from here. Besides test results, a description and commenting area are provided.</p><div class='checkbox'><label><input type='checkbox' id='dont-notify' onchange='window.dismissPermanently = this.checked'> Don't notify me anymore (permanent)</label></div>",
```